integration/tests/cook/test_multi_user.py

```diff
@@ -485,3 +485,55 @@ def test_user_total_usage(self):
                 self.assertEqual(len(job_uuids), total_usage['jobs'], total_usage)
             finally:
                 util.kill_jobs(self.cook_url, job_uuids)
+
+
+    def test_queue_quota_filtering(self):
+        bad_constraint = [["HOSTNAME",
+                           "EQUALS",
+                           "lol won't get scheduled"]]
+        user = self.user_factory.new_user()
+        admin = self.user_factory.admin()
+        uuids = []
+        default_pool = util.default_pool(self.cook_url)
+        pool = default_pool or 'no-pool'
+        def queue_uuids():
+            try:
+                queue = util.query_queue(self.cook_url).json()
+                uuids = [j['job/uuid'] for j in queue[pool] if j['job/user'] == user.name]
+                self.logger.info(f'Queued uuids: {uuids}')
+                return uuids
+            except BaseException as e:
+                self.logger.error(f"Didn't reach desired instance count: {e}")
```

Comment:
I don't understand the error message here

Reply:
Bad copy paste :(